src/listener/mod.rs

```diff
@@ -52,3 +84,50 @@ pub(crate) fn is_transient_error(e: &io::Error) -> bool {
         ConnectionRefused | ConnectionAborted | ConnectionReset
     )
 }
+
+/// Information about the `Listener`.
+///
+/// See [`Report`](../listener/trait.Report.html) for more.
+#[derive(Debug, Clone)]
+pub struct ListenInfo {
+    conn_string: String,
+    transport: String,
+    tls: bool,
```

Comment:
Maybe this should be an option over some tls version Enum?

Reply:
Oh, that's a fun one! -- It'd be nice if we could get information about the TLS version / features used -- but we probably should design that further when we introduce TLS into Tide itself. I'm thinking until then just having a bool may be good enough.